zubhub_backend/zubhub/creators/serializers.py

```diff
@@ -259,6 +261,7 @@ def get_cleaned_data(self):
 
     def save(self, request):
         creator = super().save(request)
+        creator.tags.set(self.validated_data.get('creator_tags', []))
```

Comment:
this shouldn't be done here @yokwejuste. You should take a second look at my little pointers I commented on the issue itself.

Reply:
* First the tag here should be a string and not an array of strings (I am not sure we expect more than one tag on the frontend during registration do we?)
* getting tag should be done in `get_cleaned_data` function above and set in the adapter.
* Also remember we need custom validation for the submitted tag. We can't trust the tags submitted through the api since anything can be submitted. We need to validate it both on the frontend and on the backend.
One way to validate it is to check that the submitted tag string matches the name of one of the tags we have on the backend.

Reply:
copied, all implemented